common/src/main/java/org/opensearch/ml/common/indexInsight/IndexInsightContainer.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.indexInsight;
+
+import static org.opensearch.core.xcontent.XContentParserUtils.ensureExpectedToken;
+import static org.opensearch.ml.common.CommonValue.TENANT_ID_FIELD;
+import static org.opensearch.ml.common.indexInsight.IndexInsight.INDEX_NAME_FIELD;
+
+import java.io.IOException;
+
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.common.io.stream.Writeable;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.core.xcontent.XContentParser;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class IndexInsightContainer implements ToXContentObject, Writeable {
+    private String indexName;
+    private String tenantId;
+
+    @Builder(toBuilder = true)
+    public IndexInsightContainer(String indexName, String tenantId) {
+        this.indexName = indexName;
+        this.tenantId = tenantId;
+    }
+
+    @Override
+    public void writeTo(StreamOutput out) throws IOException {
+        out.writeString(indexName);
+        out.writeOptionalString(tenantId);
+    }
+
+    public IndexInsightContainer(StreamInput input) throws IOException {
+        indexName = input.readString();
+        tenantId = input.readOptionalString();
+    }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.startObject();
+        builder.field(INDEX_NAME_FIELD, indexName);
+        if (tenantId != null) {
+            builder.field(TENANT_ID_FIELD, tenantId);
+        }
+        builder.endObject();
+        return builder;
+    }
+
+    public static IndexInsightContainer parse(XContentParser parser) throws IOException {
+        String indexName = null;
+        String tenantId = null;
+        ensureExpectedToken(XContentParser.Token.START_OBJECT, parser.currentToken(), parser);
+        while (parser.nextToken() != XContentParser.Token.END_OBJECT) {
+            String fieldName = parser.currentName();
+            parser.nextToken();
+            switch (fieldName) {
+                case INDEX_NAME_FIELD:
+                    indexName = parser.text();
```

Comment:
Is this indexName or container_name? From the PR description, it's storing container_name?

Reply:
Yes. It's container name. Already rename it to be more readable